src/index.js

```diff
@@ -3,8 +3,8 @@ var LAZY_NODE = 2
 var TEXT_NODE = 3
 var EMPTY_OBJ = {}
 var EMPTY_ARR = []
-var map = EMPTY_ARR.map
 var isArray = Array.isArray
+var isFunc = function(x) { return typeof x === "function" }
```

Comment:
@zaceno I'd prefer to avoid this function and simply inline every `typeof x === "function"` for now.

Reply:
Can do!